addons/mail/static/src/models/thread_view_topbar/thread_view_topbar.js

```diff
@@ -441,6 +444,12 @@ function factory(dependencies) {
         isMouseOverThreadDescription: attr({
             default: false,
         }),
+        /**
+         * States whether the invite button popover is open in the topbar.
```

Comment:
This should be a determines (but it doesn't matter if it's replaced by a popover field)